lib/Backends/NNPI/tests/NNPIOperatorTest.cpp

```diff
@@ -186,11 +184,10 @@ struct BlacklistInitializer {
             {"ArithMax_int64_t/0", TestBlacklist::AnyDeviceHWEngine},
             {"ArithMin_int32_t/0", TestBlacklist::AnyDeviceHWEngine},
             {"ArithMin_int64_t/0", TestBlacklist::AnyDeviceHWEngine},
-            {"ArithMul_int32_t/0", TestBlacklist::AnyDeviceHWEngine},
-            {"ArithMul_int64_t/0", TestBlacklist::AnyDeviceHWEngine},
+            {"ArithMul_int32_t/0", TestBlacklist::AnyDeviceAnyEngine},
+            {"ArithMul_int64_t/0", TestBlacklist::AnyDeviceAnyEngine},
```

Comment:
Is this a temporary regression?

Reply:
These tests where failing on HW and we updated the isOpSupported to return false.

Reply:
(Same as other) Ok, but it used to work on ICEREF. Are these going to be fixed and re-enabled for the next release, on ICEREF and on HW?

Reply:
There are no plans to support it in HW, it was removed to align with HW. 
The current code without changes returns a wrong isOpSupported value for HW.